test/e2e/constants.ts

```diff
@@ -4,7 +4,7 @@ export const STATE_PATH = "./test/e2e/test-results/states";
 export const SCREENSHOT_PATH = "./test/e2e/test-results/screenshots";
 
 export const TEST_PROJECT_NAME = "E2E - Test - " + Date.now();
-export const TEST_PROJECT_KEY = "E" + Math.floor(Math.random()*9000) + 1000;
```

Comment:
Javascript being javascript here and converting all that to string so it was always EXXXX1000